apps/prairielearn/src/middlewares/selectAndAuthzInstanceQuestion.sql

```diff
@@ -74,7 +76,6 @@ WITH
       LEFT JOIN variant_max_submission_scores AS vmss ON (vmss.variant_id = v.id)
     WHERE
       v.instance_question_id = $instance_question_id
-      AND NOT v.open
       AND NOT v.broken
```

Comment:
I almost wonder whether we should also include broken variants here? That would allow students to go back to any variant that they had seen, rather than hiding broken variants? I'm a bit torn being being transparent (showing broken variants) and keeping things simple (hiding broken variants).

Reply:
I think allowing a student to open a broken variant may cause more confusion than necessary. Besides, in some cases a broken variant will cause an issue to be created just by opening it.